ui/build/build.types.js

```diff
@@ -68,21 +67,29 @@ function convertTypeVal (type, def, required) {
   if (fallbackComplexTypeMap.has(t)) {
     if (def.definition) {
       const propDefinitions = getPropDefinitions(def.definition, required, true)
-      let lines = []
+      const lines = []
       propDefinitions.forEach(p => lines.push(...p.split('\n')))
       return propDefinitions && propDefinitions.length > 0 ? `{\n        ${lines.join('\n        ')} }${t === 'Array' ? '[]' : ''}` : fallbackComplexTypeMap.get(t)
     }
+    // uncomment to find additonal typings that probably should not be 'any[]' or `LooseDefinition`
+    // console.error(`(${++typeErrorCount}) warning: '${key}' missing 'tsType' for complex "type": "${t}"...`)
```

Comment:
Do you remember how many of this kind are there right now?

Reply:
about 258

Reply:
Maybe we can do a type of
```
export type StringOrArrayOrObject = string | any[] | LooseDictionary
```

Reply:
For style we could use a more strict type, but I prefer to address this in another PR, so we don't have to block this to wait me